### ISSUE TITLE
Explore a possibility of removing TESTING-INSTRUCTIONS.md

### ISSUE THREAD

**Issue by moon0326** [woocommerce/woocommerce-admin#7481]
We add testing instructions for a PR twice -- in PR details and TESTING-INSTRUCTIONS.md. We also get merge conflict with TESTING-INSTRUCTIONS.md very often.

I think we might be able to get rid of TESTING-INSTRUCTION.md completely.

1. Get the last release branch.
2. Compare the release branch against the main and get the list of commits that are not in the release branch.
3. Get the list of PR #s from the commit list from step #2.
4. Extract testing instructions from github.com using the PR #s.
5. Make a file so that it can be posted in Wiki.

**Comment by ilyasfoo** [comment]
Would be great to have this feature, @moon0326 ! I think there should be reservations for some use cases though, not all testing instructions should be included. Things like fixes that only a developer can test are one of them. Or a fix that's negligent like added more padding to an element. 

Would you think it's ok to also implement some way we can exempt some of the testing instructions?

**Comment by moon0326** [comment]
> Would be great to have this feature, @moon0326 ! I think there should be reservations for some use cases though, not all testing instructions should be included. Things like fixes that only a developer can test are one of them. Or a fix that's insignificant like adding more padding to an element.
> 
> Would you think it's ok to also implement some way we can exempt some of the testing instructions?

Good point! I was thinking about adding a checkbox in the PR template. Something like below.

- [ ] Include test instructions in the release
